components/country/overview-charts.js

```diff
@@ -79,9 +79,10 @@ class TestsByGroup extends React.PureComponent {
     const testCoverageByDay = testCoverage.reduce((prev, cur) => {
       prev[cur.test_day] = prev[cur.test_day] || {}
       prev[cur.test_day][cur.test_group] = cur.count
+      const allTestCount = Object.values(prev[cur.test_day]).reduce((p,c) => p+c, 0)
       if (typeof testCoverageMaxima === 'undefined'
-          || testCoverageMaxima < cur.count) {
-        testCoverageMaxima = cur.count
+          || testCoverageMaxima < allTestCount) {
+        testCoverageMaxima = allTestCount
```

Comment:
While I was at it I also fixed this issue in calculating the maxima of the testCoverage (we were only considering the maxima of the element which had the most number of measurements and not all the elements).

Reply:
:+1: